AppKit/CPWindow/CPWindow.j

```diff
@@ -198,6 +199,9 @@ var CPWindowActionMessageKeys = [
     CPView                              _toolbarView;
 
     CPArray                             _mouseEnteredStack;
+    CPArray                             _cursorUpdateStack;
+    CPArray                             _viewsWithTrackingAreas;
```

Comment:
What about using a dictionary instead of an array here ? The key could be the UID of the view and for each check we won't need to loop on the entire arrays ?